src/sass/components/_post-box.scss

```diff
@@ -1,54 +1,81 @@
-.post-box {
-  margin-bottom: 2rem;
-
-  .cover {
-    padding: 70% 25px 0 25px;
-    background: {
-      color: #f4f5f7;
-      size: cover;
-      position: center;
-    }
+.post-box-container {
+  display: block;
 
-    .details {
-      padding: 15px 20px;
-      display: flex;
-      background-color: #ffffff;
-      font-size: 14px;
+  .post-box {
+    margin-bottom: 2rem;
 
-      & > * {
-        flex: 1 1 auto;
+    .cover {
+      padding: 70% 25px 0 25px;
+      background: {
+        color: #f4f5f7;
```

Comment:
również - użyj/stwórz zmienną.

Reply:
`$post-box-cover-bg: #f4f5f7;`
`.cover {
      padding: 70% 25px 0 25px;
      background: {
        color: $post-box-cover-bg;
      }`

Zrobiłem zmienną, umieściłem w pliku ze zmiennymi. 

**Tutaj takie samo pytanie**: nazwy zmiennych robić globalne czy bardziej personalne?

Reply:
to zależy.

kolory trzymałbym w jednym miejscu, łatwiej jest wtedy nad nimi panować i uspójniać.
co do innych zmiennych np. `$element-width`, to je można definiować w konkretnym pliku skoro nie są używane (w tym samym kontekście) w innych miejscach